package.json

```diff
@@ -92,7 +92,17 @@
   },
   "remarkConfig": {
     "plugins": [
-      "@stylelint/remark-preset"
+      "@stylelint/remark-preset",
+      [
+        "remark-lint-no-undefined-references",
+        {
+          "allow": [
+            "!NOTE",
+            "!IMPORTANT",
+            "!WARNING"
+          ]
+        }
+      ]
```

Comment:
[note] See https://github.com/remarkjs/remark-lint/tree/main/packages/remark-lint-no-undefined-references

Reply:
We could allow `[…]` at a later time.